LICENSE

```diff
@@ -10,11 +10,11 @@ met:
 1. Redistributions of source code must retain the above copyright
 notice, this list of conditions and the following disclaimer.
 
-2. Redistributions in binary form must reproduce the above copyright
+1. Redistributions in binary form must reproduce the above copyright
 notice, this list of conditions and the following disclaimer in the
 documentation and/or other materials provided with the distribution.
 
-3. Neither the name of the copyright holder nor the names of its
+1. Neither the name of the copyright holder nor the names of its
```

Comment:
How come these were changed from `2.` and `3.` to `1.` and `1.`?

Reply:
It seems it was the pre-commit hooks. It should render ok, but I can roll back to 1,2,3